src/p7_pipeline.c

```diff
@@ -681,7 +757,10 @@ p7_Pipeline(P7_PIPELINE *pli, P7_OPROFILE *om, P7_BG *bg, const ESL_SQ *sq, cons
   int              Ld;               /* # of residues in envelopes */
   int              d;
   int              status;
-  
+
+  long             sq_from;        /* start location in query in amino acids */
```

Comment:
Avoid using `long` variables. Use explicit widths instead: `int64_t`, for example.